package.json

```diff
@@ -3,60 +3,65 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-	"@testing-library/jest-dom": "^5.16.3",
-	"@testing-library/react": "^12.1.4",
-	"@testing-library/user-event": "^13.5.0",
-	"@types/jest": "^27.4.1",
-	"@types/node": "^16.11.26",
-	"@types/react": "^17.0.43",
-	"@types/react-dom": "^17.0.14",
-	"react": "^18.0.0",
-	"react-dom": "^18.0.0",
-	"react-scripts": "5.0.0",
-	"typescript": "^4.6.3",
-	"web-vitals": "^2.1.4"
+    "@ethersproject/providers": "^5.6.2",
+    "@testing-library/jest-dom": "^5.16.3",
+    "@testing-library/react": "^12.1.4",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.4.1",
+    "@types/node": "^16.11.26",
+    "@types/react": "^17.0.43",
+    "@types/react-dom": "^17.0.14",
+    "@web3-react/core": "^6.1.9",
+    "@web3-react/injected-connector": "^6.0.7",
+    "eslint-plugin-react-hooks": "^4.4.0",
```

Comment:
przenieśmy to do devDependencies, bo nie jest potrzebne przy buildzie produkcyjnym :)

Reply:
Poprawione:)